scripts/src/main/resources/scripts/command/gcviewer

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env bash
+
+# autocompletion list
+if [ "${1}" = "shortlist" ]
+then
+  if [ -z "${2}" ]
+  then
+    echo "setup start help"
+  fi
+  exit
+fi
+
+if [ -n "${DEVON_IDE_TRACE}" ]; then set -vx; fi
+# shellcheck source=scripts/functions
+source "$(dirname "${0}")"/../functions
+
+GCVIEWER_HOME="${DEVON_IDE_HOME}/software/gcviewer"
+
+# gcviewer installation
+function doSetup(){
+
+  if [ -z "${GCVIEWER_VERSION}" ]
+  then
+    doEcho "Getting the latest release of GCViewer..."
+    GCVIEWER_VERSION=$(curl "https://github.com/chewiebug/GCViewer/tags" | awk -F "/tag/" '/GCViewer/ {print $2}' | sort -r | head -1 | awk -F "\">" '{print $1}')
+  fi 
+
+  do_not_extract="Not Empty"
+  doInstall "-" "${GCVIEWER_HOME}" "gcviewer" "${GCVIEWER_VERSION}" "" "" "" "" "" "" "${do_not_extract}"
+  doCreateGCViewerScript
+
+}
+
+function doStart(){
+
+  if [ ! -d "${GCVIEWER_HOME}" ]
+  then
+    doSetup
+  else
+    cd "${GCVIEWER_HOME}" || exit
+    doDevonCommand java setup silent
+    doEcho "Starting GCViewer..."
+    doRunGCViewer
+  fi
+
+}
+
+doCreateGCViewerScript(){
+
+  cd "${GCVIEWER_HOME}" || exit
+  touch gcviewer-script
+  echo -e "#!/usr/bin/env bash\n\njava -jar gcviewer*.jar&" >> gcviewer-script
+  chmod +x gcviewer-script
+
+}
+
+doRunGCViewer(){
+
+  cd "${GCVIEWER_HOME}" || exit
+  gcviewer-script
+
+}
+
+
+# CLI
+case ${1} in 
+"help" | "-h")
+  echo "Setup or run gcviewer"
+  echo
+  echo "Arguments:"
+  echo " setup                    install the latest version GCViewer on your machine."
+  echo " start                    launch GCViewer App."
+  echo
+;;
+"setup" | "s")
+  doSetup
+;;
+"start" | "")
+  doStart
+;;
+*)
+  doError "Unkown command. Call devon gcviewer to setup or start GCViewer OR devon gcviewer help for available commands"
```

Comment:
The hint "Call devon gcviewer to setup or start GCViewer OR devon gcviewer help for available commands" does not really make sense here as still arguments of gcviewer will not be passed to gcviewer and therefore still produce the same error message.
Either remove or turn gcviewer commandlet into a [wrapper](https://github.com/devonfw/ide/blob/d0165928fac059cced751473ddb55a82e2679874/documentation/cli.asciidoc#command-wrapper).